test/test_verifylib.py

```diff
@@ -936,7 +936,7 @@ def setUpClass(self):
   def tearDownClass(self):
     """Change back to initial working dir and remove temp dir. """
     os.chdir(self.working_dir)
-    log.doing(self.test_dir)
+    log.info(self.test_dir)
     #shutil.rmtree(self.test_dir)
```

Comment:
You are right. I guess had commented the cleanup out for debugging purposes. Since it's temp directories, the os should remove them at some point anyway. Nevertheless, I commented the cleanup back in (85e607b).